src/cases/views/pages/case-detail.njk

```diff
@@ -23,64 +27,80 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-full">
         <div class="govuk-body" data-testid="app-page-body">
-          {# Table 1: top-level caseData string fields #}
-          {% set rows = [] %}
-          {% for key, value in data.case %}
-            {% if value is string %}
-              {% set rows = rows.concat([[{ text: key, classes: "govuk-!-width-one-third" }, { text: value }]]) %}
-            {% endif %}
-          {% endfor %}
+          {% if data.case.caseDetails %}
+            {% for section in data.case.caseDetails.sections %}
+              <h3 class="govuk-heading-m">{{ section.title }}</h3>
 
-          {{
-            govukTable({
-              captionClasses: "govuk-table__caption--m",
-              firstCellIsHeader: true,
-              rows: rows
-            })
-          }}
+              {% if section.type == "list" %}
+                {{
+                  list({
+                    fields: section.fields,
+                    payload: data.case.payload
+                  })
```

Comment:
Should we be passing the full payload into the list component? 

Maybe better to map this in the view model code and only pass in an list of key, values into the list?  

Would make the list component more generic and wouldn't need to know the structure of the payload?